sdk/keyvault/keyvault-admin/recordings/node/aborting_keyvaultbackupclients_requests/recording_can_abort_beginbackup.js

```diff
@@ -1,5 +1,5 @@
 let nock = require('nock');
 
-module.exports.hash = "86acedb7e766cf2157a0f825ac08d930";
+module.exports.hash = "c4f5d1bb0eee1b953a06085353188825";
```

Comment:
I find it unfortunate that the recorder only updated this line. Can we skip the recordings that only affected this hash? In those cases, the existing recordings should be sufficient.